### PR TITLE
Feature/download assets

### PR DESCRIPTION
**Pull request by AlieksandraK**
Added download assets feature

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,13 +28,31 @@ created.
 
 Give that there is already a release `v1.0.10` at `myuser/myrepo`, it is possible to add assets with the following command:
 ```shell script
-github-releaser upload-assets --account myuser --repository myrepo --tagname v1.0.10 assets/*.zip
+github-releaser upload-assets --account myuser --repository myrepo --tag-name v1.0.10 assets/*.zip
```
Comment (dfurtado) [inline README.md:31]: Nice catch!

File: github_releaser/github_releaser.py [hunk 2]
```diff
@@ -225,3 +226,79 @@ def push_files(self, branch_name: str, dest_dir: str, message: str, files: List[
...
+                            spinner.write("Could not download the asset. {} (Code: {})".format(
+                                a_content.get("message", None), a_content.status_code))
+                            spinner.fail()
+                            continue
+
+                        if not path.exists(out_dir):
+                            os.makedirs(out_dir)
+
+                        dest = path.join(out_dir, assetName)
+                        with open(dest, "wb") as f:
```
Comment (dfurtado) [inline github_releaser/github_releaser.py:295]: Might need some error handling here, if it can't write the file to disk.

Comment (AlieksandraK) [reply github_releaser/github_releaser.py:295]: Pushed the change, not sure if it's something that you thought of

**Comment by dfurtado** [comment]
Closing it since this functionality has been implemented in another project.

**Pull request closed without merging**